packages/util/src/index.ts

```diff
@@ -44,6 +44,11 @@ export * from './externals'
  */
 export * from './types'
 
+/**
+ * Helper functions
+ */
+export * from './helpers'
```

Comment:
i'm still not sure if we want to add this file to the exported methods, although it would be a nice catch-all for future helper methods. @holgerd77 any thoughts?

Reply:
Also given this some thought.

I think we should definitely not just open up a whole new helper category here on the sideline, we've Util relatively well structured now and to introduce some "fall-back" category for things we are not sure doesn't seem like a very good idea to me.

Generally with this implementation from above we run into this problem we've discussed in the chat 1-2 weeks ago (@Ghorbanian sorry, this was in our internal chat, so you just couldn't follow-up on this) that with allowing both hex-prefixed strings (being seen as hex-byte-representations) and non-hex-strings (being assumed to be numbers) there is no chance to differentiate if people *want* non-hex strings to be integers or (this actually happens a lot in practice) just have forgotten the `0x` hex prefix but the input is actually a hex-byte-representation string.

This is really dangerous. With the regex implementation from above one is already catching a lot of stuff but just not all, and this would "break" respectively get ambiguous on strings like "1102" or the like.

I think we should *at least* not allow these kind of things officially in the util library when we just did the iteration of getting a lot stricter on type checking along with the v7 releases https://github.com/ethereumjs/ethereumjs-util/releases/tag/v7.0.0 for safety reasons.

Two things I can imagine here.

1. I think the functionality would fit a lot more naturally in the Util method suite as a `intToBuffer` method as a counter part to `bufferToInt` (in the `bytes` package) we already have. But this method should then really only allow integer input of various forms (`integer` itself, `integer` string, something else?). If we want to make an additional exception for the RPC use case we should add this as custom code to the `from-rpc` methods in `Block` itself (`if (hasSomeHexPrefix) use toBuffer(), else use intToBuffer)`). This might be an exception worth to make in this use case since RPC responses should be relatively well defined I guess. On the Util integration side I am still not 100% sure if this aligns well with e.g. the more general `toBuffer` method, we likely don't want to end up with 7 different flavors of these Buffer conversion methods.
2. We generally hang this lower and just add this to an internal helper function in a new `util.ts` file in the `Block` library which we should then exclude from documentation in the `typedoc.js` file.

Atm I have a slight preference for 2., but not totally sure.

WDYGT? 🤔

Reply:
Option 2 sounds like it would come with much less ambiguity. The function can keep the name `integerToHex`. We can think about making the helper function externally available if the need arises in the future. Any thoughts, @ryanio?